Cargo.toml

```
@@ -115,7 +116,7 @@ url = "2.5"
 
 # ream dependencies
 ream-account-manager = { path = "crates/account_manager" }
-ream-beacon-api-types = { path = "crates/common/beacon_api_types" }
+ream-beacon-api-types = { path = "crates/common/api_types/beacon" }
```

```suggestion
ream-api-types-beacon = { path = "crates/common/api_types/beacon" }
```

I think this fits with our convention